en/docs/learn/design-api/test-api/test-an-api.md

```diff
@@ -0,0 +1,56 @@
+# Test APIs through the publisher test console
```

Comment:
Shall we change the title to "Testing an API in the design phase" ?

Reply:
fixed